src/auth/dto/manage-team.dto.ts

```
@@ -14,9 +14,11 @@ import {
 } from 'class-validator';
 import { Type } from 'class-transformer';
 import { ApiProperty } from '@nestjs/swagger';
+import { TeamRole, TeamMemberStatus } from '../entities/team-member.entity';
 import { TeamRole } from '../enums/teamRole.enum';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'TeamRole'. Consider to delete it or rename it.

'TeamRole' is defined here:


(lint/suspicious/noRedeclare)